graylog2-web-interface/src/views/components/aggregationbuilder/DescriptionBox.jsx

```diff
@@ -12,12 +12,16 @@ import HoverForHelp from './HoverForHelp';
 export default class DescriptionBox extends React.Component {
   static propTypes = {
     children: PropTypes.node.isRequired,
-    description: PropTypes.string.isRequired,
     configurableOptions: PropTypes.node,
+    description: PropTypes.string.isRequired,
+    help: PropTypes.string,
+    style: PropTypes.object,
```

Comment:
It's just a small implementation detail, but let's talk about the style prop. All I need to do is overwriting the top margin of one DescriptionBox inside the Aggregation Builder. I had two options:
- The current way
- Adding a styled-component inside the AggregationBuilder which extends the DescriptionBox
Requires the new component + adding a className prop + an && workaround for the css, to be more specific

It's the second time we are passing the style attribute as a prop to the inner component. Is it okay to do it this way?

Reply:
Considering the drawbacks of the `styled-component` approach, passing a `style` prop for small & isolated cases is fine to me.